lib/aws/generated/health_lake.ex

```diff
@@ -0,0 +1,90 @@
+# WARNING: DO NOT EDIT, AUTO-GENERATED CODE!
+# See https://github.com/aws-beam/aws-codegen for more details.
+
+defmodule AWS.HealthLake do
+  @moduledoc """
+  Amazon HealthLake is a HIPAA eligibile service that allows customers to store,
+  transform, query, and analyze their FHIR-formatted data in a consistent fashion
+  in the cloud.
+  """
+
+  alias AWS.Client
+  alias AWS.Request
+
+  def metadata do
+    %AWS.ServiceMetadata{
+      abbreviation: "HealthLake",
+      api_version: "2017-07-01",
+      content_type: "application/x-amz-json-1.0",
+      credential_scope: nil,
+      endpoint_prefix: "healthlake",
+      global?: false,
+      protocol: "json",
+      service_id: "HealthLake",
+      signature_version: "v4",
+      signing_name: "healthlake",
+      target_prefix: "HealthLake"
+    }
+  end
+
+  @doc """
+  Creates a Data Store that can ingest and export FHIR formatted data.
+  """
+  def create_f_h_i_r_datastore(%Client{} = client, input, options \\ []) do
```

Comment:
`FHIR` can be an acronym in `aws-beam`

Reply:
https://github.com/aws-beam/aws-codegen/pull/63